packages/graphql/tests/integration/rfcs/rfc-003.test.ts

```diff
@@ -0,0 +1,212 @@
+/*
```

Comment:
None of the code for the tests in this file is actually implemented yet? Especially the unique field stuff if we're moving this out of scope? Has this file been committed accidentally?

Reply:
Yeah, it has! Thanks for pointing it out!